controllers/apply/form-router-next/index.js

```diff
@@ -51,6 +54,36 @@ function initFormRouter({
     }
 
     router.use(
+        // Populate req.body for multipart forms before CSRF token is needed
+        (req, res, next) => {
+            const isPost = req.method === 'POST';
+            const contentType = req.headers['content-type'];
+
+            // Decide if we need to use body-parser or Formidable
+            // (eg. the latter for multipart forms with files)
+            if (
+                isPost &&
+                contentType &&
+                contentType.indexOf('multipart/form-data') !== -1
+            ) {
+                const formData = new formidable.IncomingForm();
+                formData.parse(req, (err, fields, files) => {
+                    if (err) {
+                        Sentry.captureException(err);
+                    } else {
+                        req.body = fields;
+                        req.files = files;
+                    }
+                    next();
+                });
+                formData.on('error', err => {
+                    Sentry.captureException(err);
+                    next();
```

Comment:
How come we call `next` without passing the error here? Is this not a fatal error at this point?